cfme/tests/configure/test_help_menu.py

```diff
@@ -0,0 +1,11 @@
+import fauxfactory
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+def test_update_help_menu(appliance):
```

Comment:
**REQUIRED**
Please add a description for this test. http://cfme-tests.readthedocs.io/en/master/guides/documenting.html#documenting-tests